enterprise/server/cmd/ci_runner/main.go

```diff
@@ -701,9 +701,9 @@ func run() error {
 	}
 
 	if *shutdownAndExit {
```

Comment:
just as a note, in this shutdownAndExit case there is no BES stream to flush the logs to, so these logs will just sit buffered in memory - but there is only a small amount being buffered so I think it's fine to keep it like this for simplicity.